jekyll/_ccie/nomad.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: enterprise
+section: enterprise
+title: "Introduction to Nomad"
+category: [advanced-config]
+order: 9
+description: "Introduction to Nomad"
+---
+
+We chose [Nomad](https://www.hashicorp.com/blog/nomad-announcement/) as the primary job scheduler in CircleCI 2.0.
+This document will give you the basic introduction to Nomad and helps to operate Nomad Cluster in your installation.
+
+# Basic Terminologies and Architecture
+
+### Nomad Server
+Nomad Servers are the brains of the cluster. It receives and allocate jobs to Nomad clients. In CircleCI, a Nomad server is running in your service box as a Docker Container.
+
+### Nomad Client
+Nomad Clients execute jobs allocated by Nomad servers. Usually a Nomad client runs on a dedicated machine (often a VM) in order to fully take the advantage of its machine power.
+You can have multiple of Nomad clients to form a cluster and Nomad server allocates jobs to the cluster with it's scheduling algorithm.
+
+### Nomad Jobs
+Nomad Job is a specification provided by users that declares a workload for Nomad. In CircleCI 2.0, a Nomad job corresponds to an execution of CircleCI job/build. If the job/build uses parallelism, say 10 parallelism, then Nomad will run 10 jobs.
+
+### Build Agent
+Build Agent is a Go program written by CircleCI that executes steps in a job and reports the results. Build Agent is executed as the main process inside a Nomad Job.
+
+# Basic Operations
+
+This section will give you the basic guide of operating a Nomad cluster in your installation.
+
+You can use Nomad CLI in both Nomad servers and clients but we recommend running the following commands in this section from one of your Nomad clients.
```

Comment:
Still fear the uninitiated won't know how to configure their cli to talk to the right thing. They point at the ip of the services box? How is authentication done?